frontend/components/commandPalette/CommandPaletteItem.vue

```diff
@@ -0,0 +1,94 @@
+<!-- TODO: needs to handle focus / tab-to events, get rid of focus ring -->
+<template>
+  <div>
+    <NuxtLink
+      @click="handleItemClick"
+      class="flex rounded-md"
+      :to="localePath(`/${configObject.path}`)"
+    >
+      <div
+        class="group relative flex w-full items-center space-x-2 rounded-md px-2 py-1 hover:bg-light-highlight hover:dark:bg-dark-highlight"
+      >
+        <Icon
+          :name="IconMap[configObject.iconName as keyof typeof IconMap]"
+          size="1em"
+          :alt="$t(`_global.${configObject.displayName}`)"
```

Comment:
I know that this is a bit annoying, @mattburnett-repo, but we can't use variable versions of i18n keys. Not the prettiest of code, but if we use things like this, then we won't be able to track if the keys are used for not via the [i18n scripts](https://github.com/activist-org/activist/tree/main/frontend/i18n). I'll give it a quick fix 😊

Reply:
Simple as moving `_global.` to all the `displayName`s :)